tuning/trackers/aimstack_tracker.py

```diff
@@ -97,7 +97,8 @@ def __init__(self, tracker_config: AimConfig):
             information about the repo or the server and port where aim db is present.
         """
         super().__init__(name="aim", tracker_config=tracker_config)
-        self.logger = logging.get_logger("aimstack_tracker")
+        # Configure log level
+        self.logger = logging.getLogger(__name__)
```

Comment:
Please provide an example of running tuning and the log messages printed per module. THanks!

Reply:
Do you mean all the modules where we changed logging right, I should log statements and check if changes in every file works...right ?

Reply:
@anhuong 
Sharing the example of usage of root log level in different file other than `sfttrainer.py`.
In `data_type_utils.py`, I did `import logging` and did not set logger. For testing purpose wrote warning and error logs in one of the function `get_torch_dtype`

In `sfttrainer.py` main(), I set root log level (Line 502) and called the function `get_torch_dtype`. Required behaviour should be that the error message gets logged and it's happening.

Hence, removing the line `logger = logging.getLogger(__name__)` from all the files where this is globally defined, and just using `logging.error/warning/info` in those files would automatically works according to the root log-level, set in `main()` function of `sfttrainer.py`

<img width="1066" alt="image" src="https://github.com/user-attachments/assets/ccb1a9a1-ea96-4731-99bf-aefca638c55d">

Reply:
@anhuong New changes in PR are pushed. Feel free to review it whenever possible.

Reply:
@dushyantbehl Wanted to mention something.

From the file (eval_metrices.py, tracker_factory.py etc), the global declaration of `logger = logging.getLogger(__name__)`, has been removed. This is because it runs before our code of parsing cli-argument and setting root log-level in `sfttrainer.py`. Now after removal of this, all the `logging` statements in the file works as per root log-level. This sounds good to you right.

Reply:
Thanks @Abhishek-TAMU I missed this message please ignore my comments below.